app/views/catalog/_availability_indicator.html.erb

```diff
@@ -5,17 +5,18 @@
       <%= render_physical_avail_spans(doc_avail_values, service_page_link) %>
       <%= tag('br') if doc_avail_values[:physical_exists] && (doc_avail_values[:online_available] || document.url_fulltext.present?) %>
       <% if doc_avail_values[:online_available] || document.url_fulltext.present? %>
-        <span class="btn rounded-0 mb-2 online-avail-label avail-default">Online</span>
+        <%= tag.span(t('blacklight.availability.online'), class: "btn rounded-0 mb-2 online-avail-label avail-default") %>
```

Comment:
This is another example of the use of internationalization that I am not sure about. What is the benefit of using `t('blacklight.availability.online')` instead of `Online`, unless we expect to provide the catalog in other languages?